pkg/rancher-desktop/store/preferences.ts

```diff
@@ -156,8 +156,10 @@ export const actions = {
     );
     commit('SET_PREFERENCES', newPreferences);
   },
-  setWslIntegrations({ commit }: PrefActionContext, integrations: { [distribution: string]: string | boolean}) {
-    commit('SET_WSL_INTEGRATIONS', integrations);
+  setWslIntegrations({ commit, state }: PrefActionContext, integrations: { [distribution: string]: string | boolean}) {
+    const updatedIntegrations = _.merge(_.cloneDeep(integrations), state.wslIntegrations);
```

Comment:
It would be a good idea to add a comment here on why we're doing it this way. (I realize that it'd be repeating the commit message.)

Reply:
I added some comments to describe the change.